index.js

```diff
@@ -58,6 +60,30 @@ app.post("/github_push_webhook/:id", async (req, res) => {
   }
 });
 
+app.post("/github_discord_urls", async (req, res) => {
+  try {
+    const reqBody = req.body;
+    const data = await pool.query(
+      `insert into github_discord_url (discord_url) values ($1)`,
+      [reqBody.url]
+    );
+
+    const id = await pool.query(
+      `select id from github_discord_url where discord_url = $1`,
+      [reqBody.url]
+    );
+
+    const id_matching_github_and_discord = id.rows[0];
+    res
+      .status(200)
+      .json(
+        `https://v2-github-discord-api-and-me.herokuapp.com/github_push_webhook/${id_matching_github_and_discord["id"]}`
```

Comment:
construct this string outside, in a separate function. Business logic should not be inside the api code